content/tutorials/spatial_crs_coding/index.Rmd

```diff
@@ -110,25 +128,27 @@ The R packages further down, and many that depend on them, now provide a means o
 
 - **DO:**
 
-  **The _general principle_ that we recommend is: specify the CRS by using the _EPSG code_, but do so _without_ using a proj4string.**
+  **The _general principle_ that we recommend is: specify the CRS by using the _EPSG code_, but do so _without_ using a PROJ string.**
+  
+  _Note: in case you wish to define a custom CRS yourself, ideally use WKT2 or its more convenient [PROJJSON](https://proj.org/specifications/projjson.html) counterpart._
 
 - **DON'T:**
 
-  **Don't use proj4strings, such as `+init=epsg:????`, `+proj=longlat`, ... (even though that _might_ still work, continued support for it is not to be expected).**
+  **It's not longer advised to use PROJ strings to specify a CRS, such as `+init=epsg:????`, `+proj=longlat`, ... (even though that _might_ still work, their usage is discouraged).**
 
-In the below code chunks, this is demonstrated for several important geospatial R packages: **`sf`**, **`sp`** and **`raster`**.
+In the below code chunks, specifying a CRS from the EPSG database is demonstrated for several important geospatial R packages: **`sf`**, **`sp`** and **`raster`**.
```

Comment:
```suggestion
In the below code chunks, specifying a CRS with the aid of an EPSG code is demonstrated for several important geospatial R packages: **`sf`**, **`sp`** and **`raster`**.
```

Reply:
I wanted to make clear that the code is limited to CRSs that are available in the EPSG database. As opposed to other (custom) CRSs. I'll rephrase.

Reply:
de3dd4c6